components/common/Header/Header.module.scss

```diff
@@ -0,0 +1,65 @@
+.headerContainer {
+  position: sticky;
+  top: 0;
+  background-color: #fff;
+  z-index: 1;
+  padding-bottom: 20px;
+  align-items: center;
+  .header {
+    display: grid;
+    align-items: center;
+    grid-template-columns: var(--size-input) 1fr min-content min-content;
+    padding: var(--size-gap) var(--size-gutter);
+    width: 100%;
+
+    .logo {
+      align-items: center;
+      display: flex;
+      grid-area: logo;
+      height: 100%;
+      justify-content: center;
+      width: 100%;
+    }
```

Comment:
Look like we don't need any styles here except for grid-area.
- These styles are in navIcon
```css
align-items: center;
display: flex;
```
- The rest doesn't affect the UI.
```css
height: 100%;
justify-content: center;
width: 100%;
```
Consider why we need it.

Reply:
yes, seem like there was some mistakes when try merge the style from gastby to mine